src/stores/AppStore.js

```diff
@@ -415,7 +415,11 @@ export const AppStore = types
 
         if (!isLabelStream) {
           yield self.fetchActions();
-          yield self.viewsStore.fetchTabs(tab, task, labeling);
+          if (!self.SDK.settings.onlyVirtualTabs) {
+            yield self.viewsStore.fetchTabs(tab, task, labeling);
+          } else {
+            yield self.viewsStore.addView({ virtual: true }, { autosave: false });
```

Comment:
how often does `fetchData` invoked? can it possibly create a new virtual tab for some actions on the page?

Reply:
Only on init